### PR TITLE
ROX-11315: Build Scanner on UBI9

### PR DESCRIPTION
**Pull request by jvdm**
Bumping Scanner to UBI9.

This will allow Scanner to scan RHEL9 images.

Related to [ROX-11315](https://issues.redhat.com//browse/ROX-11315)

### CODE REVIEW

**Comment by jvdm** [comment]
Depends on https://github.com/openshift/release/pull/31093

File: ci-operator/config/stackrox/scanner/stackrox-scanner-jvdm-ROX-11315.yaml [hunk 1]
```diff
@@ -0,0 +1,589 @@
+base_images:
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  ubi-minimal:
+    name: ubi-minimal
+    namespace: ocp
```
Comment (RTann) [inline ci-operator/config/stackrox/scanner/stackrox-scanner-jvdm-ROX-11315.yaml:8]: this will be `stackrox` once your other PR merges

**Review by RTann — APPROVED** [review]

**Pull request merged**